SPECS-EXTENDED/kernel-rt/kernel-rt.spec

```diff
@@ -144,17 +145,19 @@ Requires:       audit
 %description tools
 This package contains the 'perf' performance analysis tools for Linux kernel.
 
-%package -n     python3-perf
+%package -n     python3-perf-%{short_name}
 Summary:        Python 3 extension for perf tools
+Requires:       %{name} = %{version}-%{release}
 Requires:       python3
 
-%description -n python3-perf
+%description -n python3-perf-%{short_name}
 This package contains the Python 3 extension for the 'perf' performance analysis tools for Linux kernel.
 
-%package -n     bpftool
+%package -n     bpftool-%{short_name}
 Summary:        Inspection and simple manipulation of eBPF programs and maps
+Requires:       %{name} = %{version}-%{release}
 
-%description -n bpftool
+%description -n bpftool-%{short_name}
```

Comment:
Why do we even need to build a separate version of these tools for each kernel? Do they have some compiled-in values, which would prevent the packages built in `kernel.spec` from working with other kernels?

Reply:
That is, indeed, a good question. I have been following the same approach as the other kernels. Do we know why we added similar `bpftools-*` and `python3-perf-*` packages for `kernel-64k` ?
https://github.com/microsoft/azurelinux/blob/5244dca9c0af53dfdd9a1ae715a39c1f6cef3db1/SPECS/kernel-64k/kernel-64k.spec#L157 ?

Reply:
I'm going to pass this question to @rlmenge.:)

Reply:
I looked into this previously and couldn't conclusively decide. Especially since some kernels have had different sources. This should be something investigated for 4.0